apps/web/screens/Adgerdir/Article.tsx

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react'
+import cn from 'classnames'
+import {
+  ContentBlock,
+  Box,
+  Typography,
+  Stack,
+  Breadcrumbs,
+  Button,
+  Link,
+  Inline,
+  Tag,
+  GridRow,
+  Text,
+  GridColumn,
+} from '@island.is/island-ui/core'
+import {
+  Query,
+  QueryGetNamespaceArgs,
+  ContentLanguage,
+  QueryGetAdgerdirPageArgs,
+  QueryGetAdgerdirPagesArgs,
+  QueryGetAdgerdirTagsArgs,
+} from '@island.is/api/schema'
+import { withMainLayout } from '@island.is/web/layouts/main'
+import {
+  AdgerdirArticles,
+  HeadWithSocialSharing,
+} from '@island.is/web/components'
+import {
+  GET_ADGERDIR_PAGE_QUERY,
+  GET_NAMESPACE_QUERY,
+  GET_ADGERDIR_PAGES_QUERY,
+  GET_ADGERDIR_TAGS_QUERY,
+} from '../queries'
+import { ArticleLayout } from '@island.is/web/screens/Layouts/Layouts'
+import { Screen } from '@island.is/web/types'
+import {
+  Content,
+  Paragraph,
+  Intro,
+  Heading,
+} from '@island.is/web/units/Adgerdir'
+import { useI18n } from '@island.is/web/i18n'
+import routeNames from '@island.is/web/i18n/routeNames'
+import { CustomNextError } from '@island.is/web/units/errors'
+import { ColorSchemeContext } from '@island.is/web/context'
+import { useNamespace } from '@island.is/web/hooks'
+
+import * as cardStyles from '@island.is/web/components/Card/Card.treat'
+
+interface AdgerdirArticleProps {
+  article: Query['getAdgerdirPage']
+  pages: Query['getAdgerdirPages']
+  tags: Query['getAdgerdirTags']
+  namespace: Query['getNamespace']
+}
+
+const AdgerdirArticle: Screen<AdgerdirArticleProps> = ({
+  article,
+  pages,
+  tags,
+  namespace,
+}) => {
+  const n = useNamespace(namespace)
+  const { activeLocale } = useI18n()
+  const { makePath } = routeNames(activeLocale)
+
+  const { items: pagesItems } = pages
+  const { items: tagsItems } = tags
+
+  const statusNames = {
+    preparing: 'Í undirbúningi',
+    ongoing: 'Í framkvæmd',
+    completed: 'Lokið',
+  }
+
+  const { estimatedCostIsk, finalCostIsk } = article
+
+  const estimatedCostFormatted =
+    estimatedCostIsk && formatNumberToKr(estimatedCostIsk)
+  const finalCostFormatted = finalCostIsk && formatNumberToKr(finalCostIsk)
+
+  const description = article.longDescription || article.description
+
+  return (
+    <>
+      <HeadWithSocialSharing
+        title={`${article.title} | Viðspyrna fyrir Ísland`}
+        description={article.description}
+        // imageUrl={article.featuredImage?.url}
+        // imageWidth={article.featuredImage?.width.toString()}
+        // imageHeight={article.featuredImage?.height.toString()}
```

Comment:
why is this commented out?

Reply:
I wanted it to remind me to add featuredImage to these adgerdir pages. I will remove them.